### ISSUE TITLE
Notifications bug

### ISSUE THREAD

**Issue by Drac1652** [ErickLimaS/anime-website#96]
Once you updated the anilist sync, the notification only showed notification count once, but since then it hasn't shown any notification

**Comment by ErickLimaS** [comment]
yeah, the notifications has to be fixed. I'll see about that too. I don't have the date for when the fix will come out yet.
